upsetplot/plotting.py

```diff
@@ -104,10 +104,12 @@ def _check_index(df):
     return df
 
 
-def _process_data(df, sort_by, sort_categories_by, subset_size, sum_over):
+def _process_data(df, sort_by, sort_categories_by, subset_size,
+                  sum_over, min_subset_size=1):
     df, agg = _aggregate_data(df, subset_size, sum_over)
     df = _check_index(df)
-
+    agg = agg[agg >= min_subset_size]
+    df = df[df.index.isin(agg.index)]
```

Comment:
I can't recall if this is strictly necessary... depends on where df is used downstream.

Reply:
`_plot_catplot` uses `df` to add catplot so if `df` contains groups that are removed in `agg` this will cause incorrect plots.

Reply:
I suppose so.